entrypoint.sh

```diff
@@ -17,8 +17,11 @@ if [ "$ENV" == "PROD" ]; then
     python3 manage.py collectstatic --no-input
 fi
 
-python3 manage.py update_integrator_permissions
-python3 manage.py compilemessages -l fr
+# Run thoses commands on any instance (PROD, DEV, DEMO...)
+if [ "$ENV" ]; then
```

Comment:
But whats the diffrence with now, it will failed if the migrations are not ran before...

For me this commit is wrong.